pkg/products/amqonline/reconciler.go

```diff
@@ -350,3 +359,64 @@ func (r *Reconciler) reconcileBlackboxTargets(ctx context.Context, installation
 
 	return integreatlyv1alpha1.PhaseCompleted, nil
 }
+
+func (r *Reconciler) reconcilePrometheusRule(ctx context.Context, client k8sclient.Client) (integreatlyv1alpha1.StatusPhase, error) {
+	monitoringConfig := config.NewMonitoring(config.ProductConfig{})
+	keycloakServicePortCount := 2
+	rule := &monitoringv1.PrometheusRule{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "rhmi-amq-online-sli",
+			Namespace: r.Config.GetNamespace(),
+		},
+	}
+
+	rules := []monitoringv1.Rule{
+		{
+			Alert: fmt.Sprintf("AMQOnlineConsoleAvailable"),
+			Annotations: map[string]string{
+				"sop_url": "https://github.com/RHCloudServices/integreatly-help/blob/master/sops/alerts_and_troubleshooting.md",
+				"message": fmt.Sprintf("AMQ-SLO-1.1: AMQ Online console is not available in namespace \"%s\"", r.Config.GetNamespace()),
+			},
+			Expr:   intstr.FromString(fmt.Sprintf("absent(kube_endpoint_address_available{endpoint=\"console\",namespace=\"%s\"}==1)", r.Config.GetNamespace())),
+			For:    "300s",
+			Labels: map[string]string{"severity": "critical"},
+		},
+		{
+			Alert: fmt.Sprintf("AMQOnlineKeycloakAvailable"),
+			Annotations: map[string]string{
+				"sop_url": "https://github.com/RHCloudServices/integreatly-help/blob/master/sops/alerts_and_troubleshooting.md",
+				"message": fmt.Sprintf("AMQ-SLO-1.3: Keycloak is not available in namespace %s", r.Config.GetNamespace()),
+			},
+			Expr:   intstr.FromString(fmt.Sprintf("absent(kube_endpoint_address_available{endpoint=\"standard-authservice\",namespace=\"%s\"}==%v)", r.Config.GetNamespace(), keycloakServicePortCount)),
+			For:    "300s",
+			Labels: map[string]string{"severity": "critical"},
+		},
+		{
+			Alert: fmt.Sprintf("AMQOnlineOperatorAvailable"),
+			Annotations: map[string]string{
+				"sop_url": "https://github.com/RHCloudServices/integreatly-help/blob/master/sops/alerts_and_troubleshooting.md",
+				"message": fmt.Sprintf("AMQ-SLO-1.5: amq-online(enmasse) operator is not available in namespace %s", r.Config.GetNamespace()),
+			},
+			Expr:   intstr.FromString(fmt.Sprintf("absent(kube_pod_status_ready{condition='true',namespace='%s',pod=~'enmasse-operator-.*'}==1)", r.Config.GetNamespace())),
```

Comment:
I see here that the query is nicely using single quotes, which means they don't need to be escaped. Would it be worth doing that for the two other queries being added above?